ui/src/containers/AppsTableContainer.js

```diff
@@ -15,22 +16,22 @@ export class AppsTableContainer extends React.Component {
   componentWillMount () {
     this.props.getApps();
   }
-  onRowClick (event, rowId, props) {
+  onRowClick (_event, rowId) {
```

Comment:
Is `_event` param required (as it's not being used) ?

Reply:
It is the first parameter in a default function, it cannot be removed.